docs/concepts/cluster-administration/device-plugins.md

```diff
@@ -88,7 +105,7 @@ runtime configurations for accessing the allocated devices. The kubelet passes t
 to the container runtime.
 
 A device plugin is expected to detect kubelet restarts and re-register itself with the new
-kubelet instance. In version 1.8, a new kubelet instance cleans up all the existing Unix sockets
+kubelet instance. In the current implementation, a new kubelet instance deletes all the existing Unix sockets
 under `/var/lib/kubelet/device-plugins` when it starts. A device plugin can monitor the deletion
```

Comment:
Our policy is to never say "current implementation" or "currently". It's too easy for those statements to become stale. So let's change this to "In version 1.9" or "In versions 1.8 and 1.9" or "In version 1.8 and later".